src/Microsoft.AspNet.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -34,9 +37,8 @@ public DateHeaderValueManager()
             : this(
                   systemClock: new SystemClock(),
                   timeWithoutRequestsUntilIdle: TimeSpan.FromSeconds(10),
-                  timerInterval: TimeSpan.FromSeconds(1))
+                  timerInterval: TimeSpan.FromMilliseconds(200))
```

Comment:
The timerInterval only affects the precision and has no effect on the performance gains.
GetDatexxx method calls will still be 2-3 times faster regardless of what is set as the timer interval.